src/config/kubeconfig.js

```diff
@@ -1,9 +1,73 @@
 'use strict';
 
-const process = require('process');
 const pth = require('path');
 const fs = require('fs');
-const k8s = require('@kubernetes/client-node');
+// TODO: Uncomment after PR https://github.com/kubernetes-client/javascript/pull/748 lands
+// const k8s = require('@kubernetes/client-node');
+
+// TODO: Delete next two functions
+// after PR https://github.com/kubernetes-client/javascript/pull/748 lands
+
+function dropDuplicatesAndNils(a) {
+  return a.reduce((acceptedValues, currentValue) => {
+    // Good-enough algorithm for reducing a small (3 items at this point) array into an ordered list
+    // of unique non-empty strings.
+    if (currentValue && !acceptedValues.includes(currentValue)) {
+      return acceptedValues.concat(currentValue);
+    } else {
+      return acceptedValues;
+    }
+  }, []);
+}
+
+// Only public for testing.
+function findHomeDir() {
+  if (process.platform !== 'win32') {
+    if (process.env.HOME) {
+      try {
+        fs.accessSync(process.env.HOME);
```

Comment:
Hmm, we should make things async, but that involves a lot of downstream changes. I guess we should do that later.

Reply:
Yes, as they say, it's easier to go from `async` to `sync` then the other way.